cmd/olclient/gov.go

```diff
@@ -210,10 +216,28 @@ func listProposals(cmd *cobra.Command, args []string) error {
 		printProposal(ps.Proposal, ps.Funds, &ps.Votes)
 		fmt.Println("Height: ", reply.Height)
 	} else { // List multiple proposals
+		pState := governance.NewProposalState(listArgs.State)
+		pType := governance.NewProposalType(listArgs.Type)
+		proposer := keys.Address(listArgs.Proposer)
+		if listArgs.State != "" {
+			if pState == governance.ProposalStateInvalid {
```

Comment:
what if the pState here is not valid and not equal to ProposalStateInvalid(0xEE)?
similar for pType

Reply:
> what if the pState here is not valid and not equal to ProposalStateInvalid(0xEE)?
> similar for pType

If pState or pType is not a valid one, then we return error. Something not correct here?